pysteps/nowcasts/interface.py

```diff
@@ -65,6 +69,9 @@ def get_method(name):
     |  lagrangian or  | this approach extrapolates the last observation       |
     |  extrapolation  | using the motion field (Lagrangian persistence)       |
     +-----------------+-------------------------------------------------------+
+    |  probability    | this approach computes a local lagrangian probability |
+    |                 | forecasts.                                            |
```

Comment:
Lagrangian_probability as name here?

Reply:
In addition, it may be helpful for the end users to explicitly state here that the forecasts are exceedance probabilities and that the output is not a forecast rainfall amount (which is the case for the other methods).

Reply:
good point! done in 4fdc677